Sources/MountebankSwift/Common/Request.swift

```diff
@@ -0,0 +1,27 @@
+public struct Request: Equatable, Codable, CustomDebugStringConvertible {
+    public let method: HTTPMethod?
+    public let path: String?
+    public let query: JSON? // TODO [String: JSON] ?
```

Comment:
Could you resolve the TODO or remove comment?

Reply:
I would like to leave it in so the code is a living todo list of what still needs to be done in other PR's. Is that OK?